### PR TITLE
fix(live provider): fix subscribe publish channel

### PR DESCRIPTION
**Pull request by self-transition**
- Fix wrong resource value in live provider's publish function
- Lint Code

### CODE REVIEW

File: src/live-provider/index.ts [hunk 1]
```diff
@@ -54,21 +63,26 @@ export const liveProvider = (globalStore: GlobalStore | GlobalStore[]): LiveProv
     stop?.();
   },
   publish: (event: LiveEvent) => {
-    const selectedGlobalStore = getSelectedGlobalStore(globalStore, event?.payload?.dataProviderName)
+    // TODO:(xingyu) by pass dataProviderName from refine hooks
```
Comment (self-transition) [inline src/live-provider/index.ts:66]: caused by 
https://github.com/refinedev/refine/issues/4943#issuecomment-1751946455

File: src/kube-api.ts [hunk 2]
```diff
@@ -584,24 +578,16 @@ export class KubeSdk {
           throw e;
         }
       }
+      const response = exist
+        ? await this.patch(spec, 'application/apply-patch+yaml')
+        : await this.create(spec);
```
Comment (huangjunjia) [inline src/kube-api.ts:583]: 移除了 try catch 之后 await 能保证不会抛出异常吗

Comment (self-transition) [reply src/kube-api.ts:583]: 之前的 catch 也是 直接 throw error 了

**Comment by self-transition** [comment]
方便 yarn link ，还是加了一下 yarn.lock

File: src/kube-api.ts (hunk 2 above)
Comment (huangjunjia) [reply src/kube-api.ts:583]: 了解了

**Review by huangjunjia — APPROVED** [review]

**Comment by huangjunjia** [comment]
LGTM

**Pull request merged**